modules/programs/rmpc.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+let
+  inherit (lib)
+    mkIf
+    mkEnableOption
+    mkPackageOption
+    mkOption
+    types
+    ;
+
+  cfg = config.programs.rmpc;
+in
+{
+  meta.maintainers = with lib.hm.maintainers; [ aguirre-matteo ];
+
+  options.programs.rmpc = {
+    enable = mkEnableOption "rmpc";
+    package = mkPackageOption pkgs "rmpc" { nullable = true; };
+    extraConfig = mkOption {
```

```suggestion
    config = mkOption {
```

If we're using this as the actual configuration, `extra` doesn't make sense.

---

Ok